plugins/UserId/Reports/GetUsers.php

```diff
@@ -61,6 +61,8 @@ public function configureView(ViewDataTable $view)
         $view->config->show_related_reports = false;
         $view->config->show_insights = false;
         $view->config->show_pivot_by_subtable = false;
+        $view->config->no_data_message = sprintf(Piwik::translate('UserId_ThereIsNoDataForThisReport'),
```

Comment:
in the link we need to add a `rel="noreferrer noopener"` (we need to add this one for all external links for security and privacy)